docs/STYLE-GUIDELINE.md

```diff
@@ -116,8 +116,8 @@ Use the mixin by `@include <mixin-name>` for example:
 
 ```css
 :root {
```

Comment:
```suggestion
:host {
```
I think there would be a change here too

Reply:
Done